package.json

```diff
@@ -15,42 +22,45 @@
     "start": "ng serve",
     "build": "ng build --prod",
     "test": "ng test",
-    "lint": "ng lint"
+    "lint": "ng lint",
+    "fullClean": "git clean -xdf && npm install",
+    "fullCleanBuild": "npm run fullClean && npm run build"
+  },
+  "dependencies": {
+    "@microsoft/applicationinsights-analytics-js": "~2.7.4",
```

Comment:
The version numbers here are the primary reason for this special branch as it's causing grief for Angular users as it's attempting to use later versions which causes TypeScript warnings / errors (depending on the tsconfig.json)

This version locks @ v2.7.4 of Application Insights -- so this version won't support the new unload functionality